web/src/Sidebar.tsx

```diff
@@ -5,12 +5,25 @@ import { combinedStatus } from "./status"
 import "./Sidebar.scss"
 import { ResourceView, TriggerMode, ResourceStatus } from "./types"
 import TimeAgo from "react-timeago"
+import { timeAgoFormatter } from "./timeFormatters"
 import { isZeroTime } from "./time"
 import PathBuilder from "./PathBuilder"
-import { timeAgoFormatter } from "./timeFormatters"
 import SidebarIcon from "./SidebarIcon"
 import SidebarTriggerButton from "./SidebarTriggerButton"
 import { numberOfAlerts } from "./alerts"
+import styled, { keyframes } from "styled-components"
+import {
+  AnimDuration,
+  Color,
+  ColorAlpha,
+  ColorRGBA,
+  FontSize,
+  Height,
+  SizeUnit,
+  Width,
+} from "./style-helpers"
+
+const moment = require("moment")
```

Comment:
i didn't actually look this up, i just blindly copy/pasted -- is this the right way to pull in another library like this?

Reply:
We use ES6 style imports, try:

```typescript
import moment from "moment"
```